da/da.md

```diff
@@ -4,7 +4,7 @@ Rollkit provides a wrapper for [go-da][go-da], a generic data availability inter
 
 ## Details
 
-`DAClient` under the hood uses a GRPC implementation of the [go-da][go-da] DA interface. Using the `DAAddress` specified in the node's config, node creates a GRPC connection to it using go-da's gprc implementation [grpc-proxy][grpc-proxy] which is then used under the hood of `DAClient` to communicate with the underlying DA.
+`DAClient` under the hood uses a json-rpc implementation of the [go-da][go-da] DA interface. Using the `DAAddress` specified in the node's config, node creates a json-rpc connection to it using go-da's json-rpc implementation [json-rpc-proxy][json-rpc-proxy] which is then used under the hood of `DAClient` to communicate with the underlying DA.
```

Comment:
I thought we were using both? 
So is it that go-da uses json-rpc and then rollkit also support grpc for other connections?

Reply:
Yep, we'll be supporting both. Will update this doc.